modules/acs-cloud-requirements.adoc

```diff
@@ -0,0 +1,63 @@
+// Module included in the following assemblies:
+//
+// * cloud_service/acscs-default-requirements.adoc
+:_mod-docs-content-type: CONCEPT
+[id="acs-cloud-requirements_{context}"]
+= General requirements for RHACS Cloud Service
+
+[role="_abstract"]
+{product-title-managed} has some system requirements that must be met before you can install it.
+
+[WARNING]
+====
+You must not install {product-title-managed-short} on:
+
+* Amazon Elastic File System (Amazon EFS). Use the Amazon Elastic Block Store (Amazon EBS) with the default *gp2* volume type instead.
+* Older CPUs that do not have the Streaming SIMD Extensions (SSE) 4.2 instruction set.
+For example, Intel processors older than _Sandy Bridge_ and AMD processors older than _Bulldozer_.
+(These processors were released in 2011.)
+====
+
+To install {product-title-managed-short}, you must have one of the following systems:
+
+* {ocp} version {ocp-supported-version} or later, and cluster nodes with a supported operating system of {op-system-first} or {op-system-base-full}.
+* a supported managed Kubernetes platform, and cluster nodes with a supported operating system of Amazon Linux, CentOS, Container-Optimized OS from Google, {op-system-first}, Debian, {op-system-base-full}, or Ubuntu.
++
+For information about supported platforms and architecture, see the link:https://access.redhat.com/articles/7045053[{product-title} Support Matrix].
+
+Cluster nodes minimum requirements:
+
+* *Architecture*: `amd64`, `ppc64le`, or `s390x`
++
+[NOTE]
+====
+Secured cluster services are supported on {ibm-power} (`ppc64le`), {ibm-z} (`s390x`), and {ibm-linuxone} (`s390x`) clusters.
+====
+* *Processor*: 3 CPU cores
+* *Memory*: 6 GiB of RAM
++
+[NOTE]
+====
+See the default memory and CPU requirements for each component and ensure that the node size can support them.
+====
+
+Persistent storage by using persistent volume claim (PVC). A PVC is strongly recommended on the secured clusters where Scanner V4 is enabled.
```

Comment:
Should this be a list item (line 44)?

Reply:
good question.... I think this needs some slight rewriting.

Reply:
+1, and a question: Are PVCs required (as in "cluster nodes minimum requirements") or recommended? I'd also suggest slightly tweaking the wording/articles used: "by using a persistent volume claim (PVC)" and "A PVC is strongly recommended if you have secured clusters with Scanner V4 enabled."